### PR TITLE
fix(test): TestConfigReloader, wait for trivy db download

### PR DESCRIPTION
**Pull request by peusebiu**
**What type of PR is this?**

**Which issue does this PR fix**:
#1525 

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — APPROVED** [review]

**Comment by rchincha** [comment]
2023-06-22T15:06:29.9602951Z   reload scrub and CVE config ✔✔✔{"level":"info","trivyJavaDownloadURL":"ghcr.io/aquasecurity/trivy-java-db","time":"2023-06-22T15:05:25.61750386Z","message":"Config: using default Trivy Java DB download URL."}
2023-06-22T15:06:29.9603788Z ✔✔✔✔{"level":"info","time":"2023-06-22T15:05:25.718576809Z","message":"config file changed, trying to reload config"}
2023-06-22T15:06:29.9604926Z {"level":"info","trivyJavaDownloadURL":"ghcr.io/aquasecurity/trivy-java-db","time":"2023-06-22T15:05:25.720307929Z","message":"Config: using default Trivy Java DB download URL."}
2023-06-22T15:06:29.9605387Z ✔✘
2023-06-22T15:06:29.9605522Z 
2023-06-22T15:06:29.9605529Z 
2023-06-22T15:06:29.9605958Z Failures:
2023-06-22T15:06:29.9606105Z 
2023-06-22T15:06:29.9606341Z   * zotregistry.io/zot/pkg/cli/config_reloader_test.go 
2023-06-22T15:06:29.9606668Z   Line 395:
2023-06-22T15:06:29.9606937Z   Expected: true
2023-06-22T15:06:29.9607217Z   Actual:   false
2023-06-22T15:06:29.9607375Z 
2023-06-22T15:06:29.9607382Z 
2023-06-22T15:06:29.9607502Z 40 total assertions

**Comment by peusebiu** [comment]
yes, I can not make any sense out of it, locally passes every time, github fails every time

File: pkg/extensions/search/cve/trivy/scanner.go [hunk 1]
```diff
@@ -373,9 +373,12 @@ func (scanner Scanner) updateDB(dbDir string) error {
 
 	registryOpts := fanalTypes.RegistryOptions{Insecure: false}
 
+	scanner.log.Debug().Str("dbDir", dbDir).Msg("Started downloading Trivy DB to destination dir")
+
 	err := operation.DownloadDB(ctx, "dev", dbDir, scanner.dbRepository, false, false, registryOpts)
 	if err != nil {
-		scanner.log.Error().Err(err).Str("dbDir", dbDir).Msg("Error downloading Trivy DB to destination dir")
+		scanner.log.Error().Err(err).Str("dbDir", dbDir).
```
Comment (rchincha) [inline pkg/extensions/search/cve/trivy/scanner.go:380]: Error msgs matter, use one scheme, start lower case camel-case strs

**Comment by rchincha** [comment]
Otherwise, lgtm

**Review by andaaron — APPROVED** [review]

**Pull request merged**